scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java

```diff
@@ -1823,12 +1825,12 @@ private void sendMessage(final RawData message, final Connection connection, fin
 	 *         be send.
 	 * @see EndpointContextMatcher#isToBeSent(EndpointContext, EndpointContext)
 	 */
-	private boolean checkOutboundEndpointContext(final RawData message, final EndpointContext connectionContext) {
+	private boolean checkOutboundEndpointContext(final RawData message, final Connection connection,
```

Comment:
The connection param is added but seems not to be used.

Reply:
I fix it.